src/js/responsive-toggler.test.ts

```diff
@@ -1,9 +1,11 @@
 import EVENTS from '@js/constants/events-map';
+import selectorsMap from '@constants/selectors-map';
 import initResponsiveToggler from './responsive-toggler';
 import initEmitter from './prestashop';
 import {
   desktopElement, mobileElement, mobileElementId, contentValue, desktopElementId,
 } from './constants/mocks/swapElements-data';
+import ThemeType = Theme.ThemeType;
```

Comment:
@ga-devfront why this import? seems useless.

Reply:
on my side then I test without this it fail.

Reply:
Maybe this import is required elsewhere.
![obraz](https://github.com/PrestaShop/hummingbird/assets/5007456/f855121e-96d8-4f03-bdb9-01aac175f06b)

Is this even a valid way of importing modules?

Reply:
https://github.com/evanw/esbuild/issues/1283